docs/custom_template/partials/footer.tmpl.partial

```diff
@@ -8,7 +8,12 @@
         <a href="#top">Back to top</a>
       </span>
       {{{_appFooter}}}
-      {{^_appFooter}}<span>Generated by <strong>DocFX</strong></span> | Copyright (c) 2018 The NUnit Project - Licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0</a>{{/_appFooter}}
+      {{^_appFooter}}<span>Generated by <strong>DocFX</strong></span> | Copyright (c) 2018-<span id="currentYear"></span> The NUnit Project - Licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0</a>{{/_appFooter}}
     </div>
   </div>
 </footer>
+
```

Comment:
I think there might be a separate place for the footer script includes. Let me confirm that

Reply:
Updated.
Here is an example by which I corrected
https://github.com/dotnet/docfx/blob/dev/src/docfx.website.themes/default/styles/main.js

made a build, everything works.